lib/simplifyExpression/basicsSearch/basicRules.js

```diff
@@ -6,22 +6,25 @@ const checks = require('../../checks');
 const defineRuleString = require('../../util/defineRuleString');
 const Node = require('../../node');
 
-// e.g. -(-3) -> 3
-const NEGATION = defineRuleString('--#a', '#a');
-
-// Simplifies two unary minuses in a row by removing both of them.
-// e.g. -(- 4) --> 4
-function negation(node) {
-  const newNode = rewriteNode(NEGATION, node);
+// Helper function to apply a rule or its reverse to the inputNode
```

Comment:
I have a feeling we'll be using this outside of the basic rules, maybe put it at the top level somewhere? or we can wait until you start on more rules and then figure it out then